sdk/ai/ai-projects/review/ai-projects.api.md

```diff
@@ -0,0 +1,1988 @@
+## API Report File for "@azure/ai-projects"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { AbortSignalLike } from '@azure/abort-controller';
+import { ClientOptions } from '@azure-rest/core-client';
+import { Paged } from '@azure/core-paging';
+import { RequestParameters } from '@azure-rest/core-client';
+import { StreamableMethod } from '@azure-rest/core-client';
+import { TokenCredential } from '@azure/core-auth';
+
+// @public
+export interface AgentDeletionStatusOutput {
+    deleted: boolean;
+    id: string;
+    object: "assistant.deleted";
+}
+
+// @public
+export interface AgentEventMessage {
+    data: AgentThreadOutput | ThreadRunOutput | RunStepOutput | ThreadMessageOutput | MessageDeltaChunk | RunStepDeltaChunk | string;
+    event: AgentStreamEventType | string;
+}
+
+// @public
+export interface AgentEventMessageStream extends AsyncDisposable, AsyncIterable<AgentEventMessage> {
+}
+
+// @public
+export interface AgentOutput {
+    created_at: number;
```

Comment:
We don't use underscores in JavaScript - is there a reason this is all using underscores?

Reply:
This is auto generated from typespec

Reply:
@MaryGao @joheredi Is there a codegen issue here then? I'd expect the JS emitter to generate idiomatic code (createdAt instead of created_at, etc)

Reply:
I think this might be an RLC thing. For RLC we don't normalize, we leave it in the wire format because we're not trying to editorialize. However, since this is a convenient client we expect it to have normalized names.